packages/cron/test/storage.spec.js

```diff
@@ -42,11 +47,16 @@ describe('cron - check user storage quotas', () => {
       percentStorageUsed: 79
     })
 
-    test3user = await createUserWithFiles(dbClient, {
-      email: 'test3@email.com',
+    testUser90percent1 = await createUserWithFiles(dbClient, {
+      email: 'testUser90percent1@email.com',
       percentStorageUsed: 90
     })
 
+    testUser90percent2 = await createUserWithFiles(dbClient, {
+      email: 'testUser90percent2@email.com',
+      percentStorageUsed: 91
+    })
+
     test4user = await createUserWithFiles(dbClient, {
```

Comment:
Can we also rename this user variable? Liked the namings you started to use as they are way more clear